editions/tw5.com/tiddlers/filters/toggle Operator.tid

```diff
@@ -0,0 +1,13 @@
+caption: toggle
+created: 20201107153758990
+modified: 20201107154300086
+op-input: a list of items
+op-output: the input list with the title  <<.place R>> toggled
+op-parameter: an item to toggle in the input list
+op-parameter-name: R
+op-purpose: toggle the title specified in the operand in the input
+tags: [[Filter Operators]] [[Listops Operators]] [[Order Operators]]
+title: toggle Operator
+type: text/vnd.tiddlywiki
+
```

Comment:
@saqimtiaz  I cannot find the position of below typo in https://tiddlywiki.com/prerelease/#toggle%20Operator

![image](https://user-images.githubusercontent.com/830394/98584183-c055de80-22da-11eb-8ede-9f09c51ae9b9.png)

The `search-replace` shall be changed by `toggle` 

Sorry I could not find the position to make a PR.

Reply:
@kookma its in a field. If you can make a PR please do or I will take care of it a little later.

Reply:
@kookma thanks #5020